src/include/utils/assert_utils.h

```diff
@@ -0,0 +1,70 @@
+//
+// Created by steven on 1/29/16.
+//
+
+#ifndef DENOVOGEAR_ASSERT_UTILS_H
+#define DENOVOGEAR_ASSERT_UTILS_H
+
+#include <cassert>
+
```

Comment:
Please wrap these functions such that if NDEBUG is defined they evaluate to no-ops.